src/app/tests/TestClusterInfo.cpp

```diff
@@ -23,9 +23,12 @@
  */
 
 #include <app/ClusterInfo.h>
+#include <app/util/mock/Constants.h>
```

Comment:
Seems this test don't need the mocked event data? Some random number / constants should be enough for testing IsEventPathSupersetOf(), IsValid...() etc?

Reply:
Prefer using the mockEventData here, ConcreteEventPath testPath(kMockEndpoint1, MockClusterId(1), MockEventId(2)); yes, without mockEventData, it should also be ok